<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRun" >
    <property name="title" >
     <string>Run</string>
    </property>
    <addaction name="actionLoad_target" />
    <addaction name="actionRun" />
    <addaction name="actionContinue" />
    <addaction name="actionStep_into" />
    <addaction name="actionStep_over" />
    <addaction name="actionStep_out" />
    <addaction name="actionPause" />
    <addaction name="actionKill" />
   </widget>
   <addaction name="menuRun" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QDockWidget" name="dwLog" >
   <property name="windowTitle" >
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPlainTextEdit" name="pteLog" >
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="tabStopWidth" >
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionRun" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Run</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionContinue" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Continue</string>
   </property>
   <property name="shortcut" >
    <string>F4</string>
   </property>
  </action>
  <action name="actionStep_into" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Step into</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionStep_over" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Step over</string>
   </property>
   <property name="shortcut" >
    <string>F6</string>
   </property>
  </action>
  <action name="actionPause" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Pause</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionLoad_target" >
   <property name="text" >
    <string>Load target</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStep_out" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Step out</string>
   </property>
   <property name="shortcut" >
    <string>F7</string>
   </property>
  </action>
  <action name="actionKill" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Kill</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+K</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
